### PR TITLE
chore: fixed projen deps

### PR DESCRIPTION
**Pull request by voho**
removes aws-cdk, etc

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -52,21 +50,19 @@
...
     "typescript": "^4.5.5"
   },
   "peerDependencies": {
-    "@aws-cdk/core": "^1.123.0",
     "constructs": "^3.2.27",
     "monocdk": "^1.123.0"
   },
   "dependencies": {
-    "@aws-cdk/core": "^1.123.0",
     "monocdk": "^1.123.0"
```
Comment (echeung-amzn) [inline package.json:66]: Need to remove this too, it should only be in peer/dev deps.

**Review by echeung-amzn — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (voho) [reply package.json:66]: I do not see it locally. Maybe i updated the PR in a wrong way. Let me check.

Comment (voho) [reply package.json:66]: Is it still there? https://github.com/cdklabs/cdk-monitoring-constructs/blob/4b9118b4104698803c300b6b20925ec07757a59c/package.json

Comment (voho) [reply package.json]: Removed it as well.

**Review by echeung-amzn — APPROVED** [review]

**Pull request merged**